src/dummy/toga_dummy/widgets/label.py

```diff
@@ -9,7 +9,7 @@ def set_alignment(self, value):
         self._set_value('alignment', value)
 
     def set_text(self, value):
-        self._set_value('text', self.interface._text)
```

Comment:
I see this pattern is used in a lot of the `text` properties across various widgets. Can you give an example of where it makes a difference, i.e. where the implementation layer needs the original object rather than the normalized value?

Reply:
After discussion, it seems like there might no longer be any examples of this. @freakboy3742 thinks it may have been made obsolete by refactoring the 0.3 branch to make the native widget the single source of truth for all user-editable properties.